webrtc-stats.html

```diff
@@ -3016,9 +3016,13 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  The URL of the TURN or STUN server indicated in the that translated this IP
-                  address. It is the URL address surfaced in an
-                  <code>RTCPeerConnectionIceEvent</code>.
+                  For local candidates this is the URL of the ICE server from
```

Comment:
Nice catch. Like relayProtocal, we should probably say "This is only present for local candidates."

Reply:
see below where i coped from relayProtocol :-)